package-lock.json

```diff
@@ -6,113 +6,163 @@
     "abbrev": {
       "version": "1.1.0",
       "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.1.0.tgz",
-      "integrity": "sha1-0FVMIlZjbi9W58LlrRg/hZQo2B8="
+      "integrity": "sha1-0FVMIlZjbi9W58LlrRg/hZQo2B8=",
+      "requires": {}
```

Comment:
wait, was your solution to unconditionally add a `requires:{}`? Mehhhh. I don't like this kinda noop crud :\